lib/vsc/filesystem/oceanstor.py

```diff
@@ -92,18 +91,19 @@
 NFS_LOOKUP_CACHE_TIME = 60
 
 # Keyword identifying the VSC network zone
-VSC_NETWORK_LABEL = 'VSC'
+VSC_NETWORK_LABEL = "VSC"
 
 
 class OceanStorClient(Client):
     """Client for OceanStor REST API"""
 
-    def __init__(self, *args, ssl_verify=True, **kwargs):
+    def __init__(self, *args, **kwargs):
         """Wrapper for Client.__init__() allowing to disable SSL certificate verification"""
         super(OceanStorClient, self).__init__(*args, **kwargs)
 
         # X-Auth-Token header
         self.x_auth_header = None
+        ssl_verify = kwargs.get('ssl_verify', False)
```

Comment:
Better default to a proper SSL connection and verify the certs
```suggestion
        ssl_verify = kwargs.get('ssl_verify', True)
```